js/ui/endSessionDialog.js

```diff
@@ -80,6 +80,24 @@ const logoutDialogContent = {
     showOtherSessions: false,
 };
 
+
+const scheduledUpdatesDialogContent = {
+    subject: C_("title", "Power Off"),
+    subjectscheduledUpdates: ("title", "You have updates scheduled to happen"),
```

Comment:
You can make this field `subject` and remove the `if (dialogContent.subjectscheduleUpdates ...)` condition below. Also, this needs to have the `C_` otherwise it's not marked for translation.